draft-ietf-tls-tls13.md

```diff
@@ -1383,10 +1383,7 @@ Structure of this message:
 
 %%% Key Exchange Messages
 
-       struct {
-           uint8 major;
-           uint8 minor;
-       } ProtocolVersion;
+       uint8 ProtocolVersionID[2];
```

Comment:
A signature algorithm is not a number. A version number is, however. The point here is that these are arbitrary identifiers, not the actual version number. Note that we do actually use the version number itself in the context strings. Calling one the version number and the other the version ID makes them explicitly separate.